pkgs/applications/science/math/pagsuite/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchzip
+, cmake
+, gmp
+, scalp
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pagsuite";
+  version = "1.80";
+
+  src = fetchzip {
+    url = "https://gitlab.com/kumm/${pname}/-/raw/master/releases/${pname}_${builtins.replaceStrings ["."] ["_"] version}.zip";
+    sha256 = "sha256-JuRuDPhKKBGz8jUBkZcZW5s2berOewjsPNR/n7kuofY=";
+    stripRoot = false;
+    postFetch = ''
+      mv $out/${pname}*/* $out
+    '';
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    gmp
+    scalp
+  ];
+
+  enableParallelBuilding = false;
+
+  meta = with lib; {
```

Comment:
`license` is required.

Reply:
The developer didn't provide the license so we must assume that it's unfree, and `unfree` is set by default in `meta`. Do we really need to explicitly specify it?

Reply:
I see, `You can use and/or modify it for research purposes (please ask for push access to make it accessable!), for commercial use, please ask for a license.` and nothing more. OK, let's keep default.

Reply:
Thanks! Set to `unfree`.